src/main/java/com/intuit/graphql/orchestrator/federation/FieldSetUtils.java

```diff
@@ -0,0 +1,28 @@
+package com.intuit.graphql.orchestrator.federation;
+
+import static java.lang.String.join;
+
+import graphql.language.Document;
+import graphql.language.Field;
+import graphql.language.OperationDefinition;
+import graphql.parser.Parser;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.apache.commons.lang3.StringUtils;
+
+public class FieldSetUtils {
+
+  private static final Parser parser = new Parser();
```

Comment:
do we have any other parser in code ?

Reply:
yup we do.  3 classes, will consolidate.